spotbugs/etc/findbugsfilter.xsd

```diff
@@ -171,6 +178,16 @@
             </annotation></attribute>
     </complexType>
 
+    <complexType name="AnnotationType">
+        <annotation>
+            <documentation></documentation>
+        </annotation>
+        <attribute name="name" type="string">
+            <annotation>
+                <documentation>The exact or regex match pattern for a java class annotation name. If the name starts with the ~ character the rest of attribute content is interpreted as a Java regular expression.</documentation>
```

Comment:
"the rest of attribute content" -> "the rest of the attribute content

Reply:
Adjusted. Note that this is a copy paste error (that still exists in the other documentation tags as well).